kikuchipy/indexing/__init__.py

```diff
@@ -25,10 +25,12 @@
 
 from kikuchipy.indexing._merge_crystal_maps import merge_crystal_maps
 from kikuchipy.indexing.orientation_similarity_map import orientation_similarity_map
+from kikuchipy.indexing._refinement import EBSDRefinement
 from kikuchipy.indexing import similarity_metrics
 from kikuchipy.indexing._static_pattern_matching import StaticPatternMatching
 
 __all__ = [
+    "EBSDRefinement",
```

Comment:
I suggest leaving `EBSDRefinement` out of the public API, as it is meant to be used via `EBSD` methods only, right? Then it should be removed from this list.